config/locales/project.en.yml

```diff
@@ -134,6 +134,16 @@ en:
     notifications:
       completed_html: <h3>Project completed<h3><p>This project was completed on %{date} and cannot be changed.</p>
       not_assigned_html: <h3>Not assigned to project</h3><p>This project is not assigned to you and cannot be changed, you can add notes or contacts if required.</p>
+    region:
```

Comment:
❤️

Reply:
Thought I'd add them now as we will definitely need them later!